internal/commands/app/diff.go

```diff
@@ -109,13 +142,13 @@ func (cmd *CommandDiff) Handler(profile *user.Profile, ui terminal.UI, clients c
 		return err
 	}
 
-	if cmd.inputs.IncludeDependencies {
-		appDependencies, err := local.FindAppDependencies(app.RootDir)
+	if cmd.inputs.IncludeNodeModules || cmd.inputs.IncludePackageJSON {
```

Comment:
and then here you'd just be adding a third ` || cmd.inputs.IncludeDependencies` (doing it this way will help the PR for `REALMC-10088` demonstrate all the places removing this was affected)